README.adoc

```diff
@@ -0,0 +1,82 @@
+= Kiali Test Mesh
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Introduction
+
+Kiali test components to be used to generate test service meshes to validate that our components are functioning as expected.
+
+=== Kiali Test Service
+
+The 'Kiali Test Service' is a simple Go based server which will allow easy routing between nodes in your service graph using only a single REST call.
+
+The main endpoint of interest is the '/route' endpoint which can be passed a 'path' query parameter containing a comma delimited list of nodes to be visited.
+
+For example, consider you are running a cluster of 3 'Kiali Test Services' where they have hostnames of 'a', 'b' and 'c'. The following URL will cause the 'a' service to connect to 'b' which then cause 'b' to connect to 'c': 'http://a/route?path=b,c'. Using similar endpoints can be used to quickly generate any type of service graph topology.
+
+=== License and Copyright
+
+See the link:./LICENSE[LICENSE file].
+
+== Building
+
+=== Building the Kiali Test Service
+
+[NOTE]
+In order to build the 'Kiali Test Service' you will need to have make and Go lang installed and available on your $PATH.
+
+To build the 'Kiali Test Service' you will need to clone the project from git into your 'GOPATH' and run the 'make' command.
+
+For example, to check 'kiali-test-mesh' into your home directory under a directory called 'kiali-test-mesh':
+
+[source,shell]
+----
+export GOPATH=~/kiali-test-mesh/GOPATH
```

Comment:
is that a typo? Having /GOPATH at the end here?

Reply:
I also prefer to see env var used - $HOME - rather than the bash style "~"

Reply:
Its not a typo, I always name the directory where my gopath is to GOPATH. It helps me remember what the actual gopath directory is suppose to be. But that is just a quirk of how I personally do things, I can rename this to how swscore is doing things.

I will update to $HOME